src/Evaluators/reduced_evaluator.jl

```diff
@@ -86,6 +86,43 @@ function ReducedSpaceEvaluator(
     )
 end
 
+function ReducedSpaceEvaluator(nlp::ReducedSpaceEvaluator, device = nothing) 
```

Comment:
do we want to use `device=nothing` as default?

Reply:
I don't know. It's in a way to force the device in the powerflow. IMO neither `CPU()` nor `CUDADevice()` is right.

Reply:
This is resolved now.